extensions/omnifocus/src/list-tasks.tsx

```diff
@@ -1,13 +1,36 @@
-import { List } from "@raycast/api";
+import { List, getPreferenceValues } from "@raycast/api";
 import { usePromise } from "@raycast/utils";
 import { useCallback, useState } from "react";
 import { listPerspectives } from "./lib/api/list-perspectives";
 import { listPerspectiveTasks } from "./lib/api/list-perspectives-tasks";
 import { TaskList } from "./lib/components/task-list";
 import { ValidateRequirements } from "./lib/components/with-requirements";
+import { GroupBy } from "./lib/components/task-list";
+
+type Preferences = {
+  showDetailsByDefault: boolean;
+  defaultPerspective?: string;
+  defaultGrouping?: GroupBy;
+};
```

Comment:
You can also remove this type, and use the `Preferences` namespace that Raycast will provide.

![CleanShot 2025-03-11 at 21 10 43@2x](https://github.com/user-attachments/assets/4d5d7525-9816-4883-b963-fba4f3512eb5)

![CleanShot 2025-03-11 at 21 11 54@2x](https://github.com/user-attachments/assets/1c089929-ba92-40af-9625-9b04f097d773)

It will use a string union instead of your `GroupBy` type, but there identical.